extensions/interactions/MusicNotesInput/static/css/musicNotesInput.css

```diff
@@ -106,7 +106,7 @@
 
 .oppia-music-input-on-staff {
   position: absolute;
-  z-index: 1000;
+  z-index: 14;
```

Comment:
Can you add a comment to warn developers why this shouldn't be changed in future (or what constraints they should be careful about here)? I assume you changed it for a reason.